src/components/AddEvent/AddEvent.js

```diff
@@ -12,103 +13,144 @@ import {
   Typography,
 } from "@mui/material";
 
+import { initialValues, validationSchema } from "../../utils/utils";
 import TextFieldWrapper from "./FormsUI/TextField";
 import DateTimePicker from "./FormsUI/DateTimePicker";
-
-import { initialValues, validationSchema } from "../../utils/utils";
+import { EventsContext } from "../../EventsContext";
+import { SnackbarContext } from "../../contexts/SnackbarContext";
+import { AttendeesInput } from "./FormsUI/AttendeesInput";
 
 const AddEvent = () => {
+  const { createEvent } = useContext(EventsContext);
+  const { setSnackData } = useContext(SnackbarContext);
+  const [selectedAttendees, setSelectedAttendees] = useState([]);
+
+  let navigate = useNavigate();
+
+  const updateSnack = (message, open, severity) => {
```

Comment:
I think this function is useless. It just calls another function with a different name but same parameters. You can call directly setSnackData wherever you need to.